### PR TITLE
Update events.json

### PR DESCRIPTION
**Pull request by ananya**
Issue Number #70 
Changes proposed in this pull request : added more events
[Live Link](provide link here)

### CODE REVIEW

File: static/js/events.json
Comment (gauthamzz) [inline static/js/events.json]: where is hack in the north
where is opencode 😄

**Comment by Anupam-dagar** [comment]
@ananya-agrawal resolve merge conflicts

**Comment by ananya** [comment]
@Anupam-dagar image was added for contribution section.

**Review by gauthamzz — CHANGES_REQUESTED** [review]
Didn’t I already merge this

**Comment by ananya** [comment]
@gauthamzz there were two pr containing different events . HINT was added in the earlier one and added opencode in this. Maybe while fixing conflicts two events got repeated so i will verify that after this pr gets merged . Tell if some changes are to be made.

File: static/js/events.json
Comment (Anupam-dagar) [inline static/js/events.json:324]: proper indentation needed, everything else LGTM :+1:

**Review by Anupam-dagar — COMMENTED** [review]
indentation needed

**Pull request closed without merging**
